Automattic/jetpack
# Masterbar: Make Desktop Switcher look the same between Calypso and wp-admin

#### Changes proposed in this Pull Request:

* Replaces the screen setting version with a new screen meta link
* Adds a JS handler to move the screen meta link in the right position and toggle the view selector

#### Does this pull request change what data or activity we track or use?


No

#### Testing instructions:






* Install [Jetpack Beta](https://jetpack.com/download-jetpack-beta/) on a test site and switch to this branch.
* Go to /wp-admin/themes.php and look for a "View" screen meta link in the top right of the screen.
* Click the tab and makes sure the selector opens.
* Select an option and verify that the Default selection takes you to the Calypso screen.

#### After

<img width="243" alt="image" src="https://user-images.githubusercontent.com/1398304/153927817-dee6dceb-9559-46cb-91af-5a5d9c1169ee.png">
